### PR TITLE
Dashulik branch

### PR DESCRIPTION
**Pull request by Murzik1234**

### CODE REVIEW

File: objects/triangle1.obj
Comment (konchaStar) [inline objects/triangle1.obj:1]: What is that???????

File: src/main/java/com/example/_3dgraphics/Camera.java [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package com.example._3dgraphics;
-
+import com.example._3dgraphics.math.Vec4d;
 import com.example._3dgraphics.math.Matrix4x4;
 import com.example._3dgraphics.math.Vec4d;
 
+
```
Comment (konchaStar) [inline src/main/java/com/example/_3dgraphics/Camera.java:6]: Remove empty line

File: src/main/java/com/example/_3dgraphics/ZBuffer.java
Comment (konchaStar) [inline src/main/java/com/example/_3dgraphics/ZBuffer.java:1]: No need to use class, use simple array instead!!!!!!!!!!

**Review by konchaStar — APPROVED** [review]

File: objects/triangle1.obj
Comment (Murzik1234) [reply objects/triangle1.obj:1]: Fuck you:)

File: src/main/java/com/example/_3dgraphics/Camera.java (hunk 1 above)
Comment (Murzik1234) [reply src/main/java/com/example/_3dgraphics/Camera.java:6]: remove empty line by yourself, bitch!!!!!!!

File: objects/triangle1.obj
Comment (konchaStar) [reply objects/triangle1.obj:1]: Kicked ;)

File: src/main/java/com/example/_3dgraphics/Camera.java (hunk 1 above)
Comment (konchaStar) [reply src/main/java/com/example/_3dgraphics/Camera.java:6]: Get the fuck off you dumb peice of shit!!!!!

Comment (Murzik1234) [reply src/main/java/com/example/_3dgraphics/Camera.java:6]: I fucked your mother twice!

**Pull request merged**